src/scripts/main.js

```diff
@@ -5,4 +5,26 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+people.forEach((person) => {
+  const age = person.died - person.born;
+  const century = Math.ceil(person.died / 100);
+
+  if (person.sex === 'f') {
+    person.sex = 'Female';
+  }
```

Comment:
```suggestion
  if (person.sex === 'f') {
    person.sex = 'Female';
  }
```
Коли ти пишеш `person.sex = 'Female';` ти змінюєш властивість `sex` у вхідному об'єкті. Нам цього робити не потрібно. ТОбі треба відображати інформацію в таблиці відповідно до цього поля, а не міняти його.
Зроби, як я уже писав у коментарі вище:

`<td>${(person.sex === 'm') ? 'Male' : 'Female'}</td>`

Reply:
Привіт, я так і зробив. Як мені чекнути що запушилось?